library/private/options_tools.h

```diff
@@ -206,7 +206,7 @@ color_t parse(const std::string& str)
 //----------------------------------------------------------------------------
 /**
  * Parse provided string into a direction_t.
```

Comment:
do you think we should improve format as well ?

Reply:
you mean detect if arbitrary x,y,z values can be formatted as `[±X][±Y][±Z]`?

Reply:
Yes, this is what I mean

Reply:
should be doable, you want it in this PR?

Reply:
as you prefer